sql/moz-fx-data-experiments/monitoring/experimenter_experiments_v1/query.py

```diff
@@ -158,8 +158,8 @@ def to_experiment(self) -> "Experiment":
             experimenter_slug=None,
             type="v6",
             status="Live"
-            if self.endDate
-            and self.endDate <= pytz.utc.localize(datetime.datetime.now())
+            if (self.endDate is None or (self.endDate
+            and self.endDate <= pytz.utc.localize(datetime.datetime.now())))
```

Comment:
Live experiments might not have an endDate set.

Reply:
Copying my feedback from slack:
> I don't have great familiarity with the experimenter API, but this seems logical to me (that live experiments may not have an end date or that the end date is in the future).

LGTM